astroid/nodes/scoped_nodes.py

```diff
@@ -3045,3 +3103,12 @@ def _get_assign_nodes(self):
             child_node._get_assign_nodes() for child_node in self.body
         )
         return list(itertools.chain.from_iterable(children_assign_nodes))
+
+    def __dump__(self, dumper):
+        data = super().__dump__(dumper)
```

Comment:
Is this a cultural reference to dumb and dumber ;) ?

Reply:
Haha no, but we can pretend it is.
I originally was passing the remap around but thought that was kind of dirty, so wanted to pass a function that did the dumping and remembered the remap so "dumper" was the first word that came to mind 😂